contracts/ITablelandControllerV1.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "./ITablelandController.sol";
```

Comment:
including this here will unnecessarily bloat the user's contract (a contract that implements V1).

since you're not creating explicit mappings between a `Policy` and a `PolicyV1`, i don't see the benefit of nesting `Policy` in `ITablelandController`. 

my suggestion was to nest a `PolicyV1` under `ITablelandControllerV1`, and then in `TablelandTables` write a function that explicitly converts a `PolicyV1` to a `Policy`. of course, this would just copy the params in this case since `Policy` hasn't changed. 

alternatively, we accept that controllers and policies will be versioned separately. in this case, i would remove `Policy` from `ITablelandController` and just import it into `ITablelandController` and `ITablelandControllerV1`. then, say we have a future `ITablelandController` that needs a new type of `Policy`... at this point we'd move `Policy` to `PolicyV1` and create a mapping from `PolicyV1` to `Policy`. then we'd have this mapping:

- `ItablelandController` (current version) -> `Policy` (current version)
- `ItablelandControllerV1` -> `PolicyV1`
- `ItablelandControllerV2` -> `PolicyV1`

so, the controller and policy versions wouldn't _necessarily_ match. it's a tradeoff. perhaps we never need to version `Policy`, in which case it would be a shame to waste contract size / gas to do the conversion.

considering that in either case, users won't have to touch the versioning scheme, i think the mental complexity in having separating the versionings trumps the downside of added contract size / gas. 

so, circling back, i would simply move `Policy` out of `ITablelandController`. hopefully that makes sense... trying to be extra verbose here.

Reply:
I think this makes sense. The Policy struct and Controller interface are related but distinct so I think it's fine to version them separately. Plus, gas savings is always nice. I've updated the PR.